assets/tasks/common.json

```diff
@@ -1351,12 +1310,13 @@
             "dependsOrder": "sequence",
             "problemMatcher": "$msCompile",
             "icon": {
-                "id": "flame",
+                "id": "repo-push",
                 "color": "terminal.ansiYellow"
             }
         },
         {
             "label": "push-release-arm64",
+            "detail": "These tasks will build the application based in the production Dockerfile and push the image to the DockerHub.",
```

Comment:
This task will build the application based on the production Dockerfile and push the image to the DockerHub.